.github/PULL_REQUEST_TEMPLATE/data_plane_template.md

```diff
@@ -47,13 +47,13 @@ The [Azure API Review Board](https://aka.ms/azsdk/onboarding/restapischedule) is
 ### Tooling
 
  * [Open API validation tools](https://aka.ms/swaggertools) were run on this PR. Go here to see [how to fix errors](https://aka.ms/ci-fix)
- * [Spectral Linting](https://aka.ms/style)
+ * [Spectral Linting](https://aka.ms/azapi/style)
```

Comment:
It may better link to https://github.com/Azure/azure-api-style-guide/blob/main/README.md, not sure if we had an aka for it.

Reply:
Agree.

Reply:
The linked doc has a header of `OpenAPI Style Guidelines` but here it is called `Spectral Linting`.